user-docs/pupil-player.md

```diff
@@ -38,7 +38,7 @@ Drag the recording directory (the triple digit one) directly onto the app icon *
 #### Workflow
 
 ```bash
-Running from source?
+# Running from source?
```

Comment:
Since this is a `bash` code block and "Running from source?" is a comment not a command. `#` indicates line comments in `bash.`